src/schema/schema_create.c

```diff
@@ -177,6 +179,10 @@ __create_file(
                 }
                 WT_ERR(__wt_strndup(session, cval.str, cval.len, &filemeta));
                 filecfg[2] = filemeta;
+                WT_ERR(__wt_reset_blkmod(session, config, buf));
+                filecfg[3] = buf->mem;
+                __wt_verbose(
+                  session, WT_VERB_BACKUP, "Config after reset_blkmod %s", (char *)buf->mem);
```

Comment:
If you intend to keep this message for merging the string should be more informative about what/where we are in the code.

Reply:
removed